js/src/ui/Form/AutoComplete/autocomplete.js

```diff
@@ -16,11 +16,24 @@
 
 import React, { Component, PropTypes } from 'react';
 import keycode from 'keycode';
-import { MenuItem, AutoComplete as MUIAutoComplete } from 'material-ui';
+import { MenuItem, AutoComplete as MUIAutoComplete, Divider as MUIDivider } from 'material-ui';
 import { PopoverAnimationVertical } from 'material-ui/Popover';
 
 import { isEqual } from 'lodash';
 
+// Hack to prevent "Unknown prop `disableFocusRipple` on <hr> tag" error
+class Divider extends Component {
```

Comment:
Split components into their own files. Started doing that as cleanups where I've found them, the 100's of lines in a file just makes it very difficult to maintain.